knimin/tests/test_ag_pulldown.py

```diff
@@ -73,6 +73,11 @@ def test_post(self):
                        ' for file download. It may take a while with many '
                        'barcodes.</h3>'), response.body)
 
+        data = {'external': 'ab', 'external': 'cd'}
```

Comment:
Can you access through the interface and add some prints? I'm not fully familiar with that part of the code so I'm unsure on how the parameters are actually received from the HTML. There may be something fancy happening there and/or a bug somewhere. I don't think we ever had 2 external surveys so there is a high probability that there is a bug in there and we never caught it...